src/go/pt-k8s-debug-collector/dumper/dumper.go

```diff
@@ -166,12 +166,12 @@ func (d *Dumper) DumpCluster() error {
 				continue
 			}
 			location = filepath.Join(d.location, ns.Name, pod.Name, "/pt-summary.txt")
-			component := d.crType
-			if d.crType == "psmdb" {
+			component := resourceType(d.crType)
+			if component == "psmdb" {
 				component = "mongod"
 			}
```

Comment:
Variable name is `component` but the function is returning a resource type and that variable is being used only once.
We could replace it with a short if statement.

```suggestion
```
```suggestion
            if rt:= resourceType(d.crType); rt == "psmdb" {
                component = "mongod"
            }

```